sklearn/feature_extraction/tests/test_text.py

```diff
@@ -562,25 +562,38 @@ def test_count_vectorizer_max_features():
 
     cv_1 = CountVectorizer(max_features=1)
     cv_3 = CountVectorizer(max_features=3)
+    cv_all = CountVectorizer(max_features=1.0)
+    cv_half = CountVectorizer(max_features=0.5)
     cv_None = CountVectorizer(max_features=None)
 
     counts_1 = cv_1.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
     counts_3 = cv_3.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
+    counts_all = cv_all.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
+    counts_half = cv_half.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
     counts_None = cv_None.fit_transform(JUNK_FOOD_DOCS).sum(axis=0)
 
     features_1 = cv_1.get_feature_names()
     features_3 = cv_3.get_feature_names()
+    features_all = cv_all.get_feature_names()
```

Comment:
Can you please check for equality and subset between the different sets of feature names. (i.e. 1 subset of 3 which is subset of None; 3 = .5, None = 1)